backend/testUtils/testSession.ts

```diff
@@ -0,0 +1,9 @@
+export const mockTestSession = {
+    test: "1",
+    teacher: "2",
+    school: "3",
+    grade_level: 4,
+    results: [],
```

Comment:
When we create a test session, would we supply an empty array for `results`? And if that's the case, what would the type for `results` be? Cause right now as `results: [Result]`, it is requiring at least one Result object @joyce-shi

Reply:
yes! we should start the array as empty and then populate it as student results are received. given this, do you mind updating the `Result` schema so the the field is optional? (I think just set `required: false`)